### DISCUSSION TITLE
After restarting the computer, the snap camera cannot be read by other apps and displays a yellow screen

### DISCUSSION THREAD

**Discussion by chase-chai** [ptrumpis/snap-camera-server#65] [Q&A] [answered]
Help! My snap camera actually worked after it was installed as directed, which is pretty cool!

However, I found that every time after restarting the computer, there would be problems, snap could not be read by other applications, for example, after selecting snap camera in OBS, it would load for a period of time and become yellow screen, but after I re-installed snap camera in pkg, it could be read again, until the next time after restarting the computer, it could not be read again

I have a mac snap camera 1.21, what causes this situation and how to solve it? Thank you so much!

**Comment by ptrumpis** [comment, marked as answer]
This might be helpful
[Resolving macOS Webcam Issues](https://github.com/ptrumpis/snap-camera-server/wiki/Resolving-macOS-Webcam-Issues)
